CHARTER.md

```diff
@@ -31,13 +31,65 @@ The membership profile is…
 ### 2.2 Community Feedback
 
 
-## 3. Meetings
+## 3. Communication
 
-### 3.1 Cadence
+### 3.1 Channels
 
-### 3.2 Annual Activities 
+Asynchronous public communication is preferred for working group activities. This section attempts to identify the communication channels along with recommendations for how and when we'll use them.
 
-Ex: budget/finance, meeting planning, elections, etc…
+*Mailing List*
+- monthly newsletter summarizing working group activities
+- used by managers for any working group announcements
+- low volume
+
+*GitHub*
+- **GitHub Discussions** used for working group feedback, suggestions, and polls
+- **GitHub Issues** used to track working group tasks
+- **GitHub Projects** used to track multiple tasks into an outcome-driven kanban aligned with working group strategic goals
+- **GitHub Pull Requests** provide opportunity for feedback from participants on any working group operating document changes
+
+*Discord*
+- use the #resource-aggregation channel for working group specific chat
+- agenda and summary of any working group calls
+- higher frequency of updates to call attention to activity from other channels
+
+*Zoom*
+- see the next section for more about working group calls
+
+### 3.2 Working Group Calls
+
+Real-time communication will utilize the community calendar to announce events to be held using Zoom for teleconferencing.
+
+*Agenda*
+- agenda is set by managers and shared a reasonable amount of time ahead of a call in Discord along with a call for topics from the community
+
+*Minutes*
+- the agenda and meeting minutes should be recorded in the GitHub repository by managers within a reasonable amount of time following the call
```

Comment:
Some projects use HackMd and other similar services.
Not sure we can have one, but tagging @anajsana just in case

Reply:
for the general community calls, agenda and minutes were being recorded in github discussions which seems like a nice way to keep it all together:
https://github.com/DevRel-Foundation/governance/discussions/categories/meeting-notes

Reply:
Resolving as discussed in our recent call: #21